### PR TITLE
tests: ensure network-bind and core-support are connected

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

File: tests/main/classic-ubuntu-core-transition/task.yaml [hunk 1]
```diff
@@ -78,3 +78,7 @@ execute: |
     wait_for_service snap.test-snapd-python-webserver.test-snapd-python-webserver
     echo "Ensure the webserver is working after a snap restart"
     curl http://localhost | MATCH "XKCD rocks"
+
+    echo "Ensure interfaces are connected"
+    snap interfaces | MATCH ":core-support.*core:core-support-plug"
+    snap interfaces | MATCH ":network-bind.*core"
```
Comment (niemeyer) [inline tests/main/classic-ubuntu-core-transition/task.yaml]: Why aren't those two exactly alike?

Comment (mvo5) [inline tests/main/classic-ubuntu-core-transition/task.yaml]: I think the should be exactly alike, the test is currently failing the network-bind is not connected and we still don't know why. This is under investigation. I will fix the test!

**Comment by fgimenez** [comment]
@mvo5 thanks for this! I think that the network-bind test should be covered in 3145 https://github.com/snapcore/snapd/pull/3145/files#diff-23e6c1e2bbf5a1a24798303971f19b99R74, is that correct? with the changes in that branch the check works just fine. The core-support verification  in this PR is not done elsewhere afaik.

**Comment by mvo5** [comment]
@fgimenez 3145 fixes the error after it has happened, i.e. it fixes the now disconnected core snap. What I'm curious about is why we have a disconnected network-bind plug in the first place. Also why does core-support connects and network-bind does not?

**Comment by mvo5** [comment]
Superseed in favour of https://github.com/snapcore/snapd/pull/3167

**Pull request closed without merging**